torat_client/netclient.go

```diff
@@ -36,7 +37,18 @@ func NetClient() {
 	log.Println("NetClient")
 	initServer()
 	var conf tor.StartConf
-	conf = tor.StartConf{ProcessCreator: embedded.NewCreator()}
+
+	tmp_dir, err := ioutil.TempDir("", "")
+	if err != nil {
+		log.Println("[NetClient] [!] Could not create temp dir for Tor: ", err)
+		return
+	}
+
+	conf = tor.StartConf{
```

Comment:
This can be `conf := ....` then the var decl above can be removed

Reply:
okay, I can change this but just so you know thats how you wrote it originally. 

I totally agree though, `:=` is better than `var` and `=`